src/main/data/Rua_copy.java

```diff
@@ -0,0 +1,119 @@
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+
+
+
+public class Rua_copy {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        String greeting = "____________________________________________________________\n" +
```

Comment:
Maybe you could consider adding access modifiers for 'greeting' and 'goodbye'. 

You could also add 'final' since these Strings will not be changed.
```suggestion
        private final String greeting = "____________________________________________________________\n" +
```